themes-default/slim/views/layouts/main.mako

```diff
@@ -134,7 +134,7 @@
             if (!window.app) {
                 console.info('Loading Vue with router since window.app is missing.');
                 const router = new vueRouter({
-                    base: document.getElementsByTagName('base')[0].getAttribute('href'),
+                    base: '${app.WEB_ROOT}/',
```

Comment:
Shouldn't adding python... Let's just fix this in js.
```
Try using `window.baseUrl.replace('/api/v2', '')`
```

Reply:
That may work, but that's not what I was going for.baseUrl is the relative baseUrl to the domain.